jetstream/tests/integration/test_analysis_integration.py

```diff
@@ -124,18 +148,19 @@ def test_metrics(self, monkeypatch, client, project_id, static_dataset, temporar
             ORDER BY enrollment_date DESC
         """
         )
-
+        a_value = "0" if analysis_unit == AnalysisUnit.PROFILE_GROUP.value else "a"
+        b_value = "0" if analysis_unit == AnalysisUnit.PROFILE_GROUP.value else "b"
```

Comment:
Is this a way to modify the underlying data to make it obvious exactly which aggregation was run? If we aggregated to profile level, the analysis IDs should be of the form `xxx0` but for client, it should be `xxxx`?

Reply:
I'm not sure I understand what you're asking. Do you mean for production data or for the tests? I don't think we'd want to modify production data in the analysis tables, and I think the test data is already of the form you describe.

Reply:
Yeah, asking about the form of the test data. I think you made the test data of this form for this reason, but just wanted to confirm

Reply:
Oh sheesh I misread "Is this a way..." as "Is there a way...", sorry!

The difference was so that I didn't accidentally get passing tests if it used the wrong analysis unit -- I just wanted them to be different in some way and this was an easy way to accomplish that.